examples/src/bin/server_acceptor.rs

```diff
@@ -117,17 +117,27 @@ fn main() {
         // connection.
         let accepted = loop {
             acceptor.read_tls(&mut stream).unwrap();
-            if let Some(accepted) = acceptor.accept().unwrap() {
-                break accepted;
+
+            match acceptor.accept() {
+                Ok(Some(accepted)) => break accepted,
+                Ok(None) => continue,
+                Err((e, mut alert)) => {
+                    alert.write(&mut stream).unwrap();
+                    panic!("error accepting connection: {}", e);
```

Comment:
Nit: prefer inline format args? (Also in the other hunk, below.)

Reply:
Good call, fixed in both spots.